engine/battle/core.asm

```diff
@@ -2710,6 +2710,14 @@ SelectMenuItem:
 	jp nz, SelectMenuItem_CursorDown ; down
 	bit 2, a
```

Comment:
Can you update the `bit 2` and `bit 1` checks to also use constants (in pokered too)?